pkg/services/mysqldb/catalog.go

```diff
@@ -131,5 +131,164 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 				},
 			}),
 		),
+		// vm only service
+		service.NewService(
+			&service.ServiceProperties{
+				ID:          "30e7b836-199d-4335-b83d-adc7d23a95c2",
+				Name:        "azure-mysqldb-vm-only",
+				Description: "Azure Database for MySQL - Server VM Only (Experimental)",
+				Metadata: &service.ServiceMetadata{
+					DisplayName: "Azure Database for MySQL (Server VM Only)",
+					ImageUrl: "https://azure.microsoft.com/svghandler/mysql/" +
+						"?width=200",
+					LongDescription: "A fully managed MySQL database service for " +
+						"app developers (Experimental)",
+					DocumentationUrl: "https://docs.microsoft.com/en-us/azure/mysql/",
+					SupportUrl:       "https://azure.microsoft.com/en-us/support/",
+				},
+				Bindable:       false,
+				Tags:           []string{"Azure", "MySQL", "Server", "VM"},
+				ChildServiceID: "6704ae59-3eae-49e9-82b4-4cbcc00edf08",
+			},
+			m.vmOnlyServiceManager,
+			service.NewPlan(&service.PlanProperties{
+				ID:          "3f65ebf9-ac1d-4e77-b9bf-918889a4482b",
+				Name:        "basic50",
+				Description: "Basic Tier, 50 DTUs.",
+				Free:        false,
+				Extended: map[string]interface{}{
+					"skuName":        "MYSQLB50",
+					"skuTier":        "Basic",
+					"skuCapacityDTU": 50,
+					"skuSizeMB":      51200,
```

Comment:
fyi only... this bothers me. I feel like the sku name "MYSQLB50" already conveys the tier and the number of DTUs. But, afaik, we _need_ to provide all of these values when provisioning via ARM. (Or that was true at one time.) Separate from this PR, we should experiment with this and maybe even talk to the MySQL team to see if they can / plan to streamline this. Picking a sku for other services isn't this difficult.